mmv1/products/bigqueryreservation/terraform.yaml

```diff
@@ -23,6 +23,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "reservation"
         vars:
           name: 'my-reservation'
+  CapacityCommitment: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{name}}"
+    import_format: ["{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "bigquery_reservation_capacity_commitment_basic"
```

Comment:
a pattern i've seen is to have a test version (`skip_docs: true`) vs a docs version (`skip_tests: true`). Would that be reasonable here?

Reply:
SGTM - will update